packages/url-state-provider/src/index.test.js

```diff
@@ -110,199 +104,59 @@ describe("currentState", () => {
     })
 
     it("should modify state search param in URL", () => {
-      var newState = provider.encode({
+      const newState = provider.encode({
         consumer1: { p: "/about", o: { tab: 1 } },
         consumer2: { p: "/items/10", o: { tab: 2 } },
       })
 
-      expect(window.location.href.indexOf(newState) >= 0).toEqual(true)
+      expect(window.location.href.includes(newState)).toBe(true)
     })
 
     it("should not modify other search params in URL", () => {
-      var newState = provider.encode({
+      const newState = provider.encode({
         consumer1: { p: "/about", o: { tab: 1 } },
         consumer2: { p: "/items/10", o: { tab: 2 } },
       })
 
-      expect(window.location.href).toEqual("http://localhost/?test1=test1&__s=" + newState + "&test2=test2")
+      expect(window.location.href).toBe("http://localhost/?test1=test1&__s=" + newState + "&test2=test2")
     })
 
     describe("search params are empty", () => {
       beforeAll(() => {
-        jest.resetModules()
+        vi.resetModules()
         delete window["__url_state_provider"]
         window.location.href = "http://localhost"
         provider.push("consumer1", { p: "/about" })
       })
 
       it("should add ?", () => {
-        var newState = provider.encode({
+        const newState = provider.encode({
           consumer1: { p: "/about" },
         })
 
-        expect(window.location.href).toEqual("http://localhost/?__s=" + newState)
+        expect(window.location.href).toBe("http://localhost/?__s=" + newState)
       })
     })
 
     it("support 50 states with a path length of 1000 characters", () => {
-      let stateCount = 50
-      let pathLength = 1000
-      let states = {}
+      const stateCount = 50
+      const pathLength = 1000
+      const states = {}
       for (let i = 0; i < stateCount; i++) {
-        var key = "consumer" + i
-        var state = {
+        const key = "consumer" + i
+        const state = {
           p: new Array(pathLength + 1).join("x"),
           o: { tab: 2, option1: "test", option2: "test " },
         }
         provider.push(key, state)
         states[key] = state
       }
 
-      var urlState = new URL(window.location.href).searchParams.get("__s")
-      // The browsers allow 2040 characters long URLs.
-      // If we stay below 1500 characters, we can still support 50 different states with
-      // a length of up to 1000 characters per path.
-      expect(urlState.length < 1500).toEqual(true)
-    })
-  })
-
-  describe("replace", () => {
```

Comment:
Did you delete these tests on purpose? They seem relevant still.

Reply:
no, by accident :(
I've reverted the deletion and adopted the tests :)